cocos/animation/marionette/animation-blend-1d.ts

```diff
@@ -53,10 +55,12 @@ export class AnimationBlend1D extends AnimationBlend {
         return that;
     }
 
-    public [createEval] (context: MotionEvalContext) {
-        const evaluation = new AnimationBlend1DEval(context, this, this._items, this._items.map(({ threshold }) => threshold), 0.0);
+    public [createEval] (context: AnimationGraphLayerWideBindingContext, clipOverrides: ReadonlyClipOverrideMap | null) {
+        const evaluation = new AnimationBlend1DEval(
+            context, clipOverrides, this, this._items, this._items.map(({ threshold }) => threshold), 0.0,
+        );
         const initialValue = bindOr(
-            context,
+            context.up,
```

Comment:
`up` is not so easy to understand.

Reply:
outerContext: https://github.com/cocos/cocos-engine/pull/13503/commits/a5080fb17a413877ebdb87748fd4bf1be66908fb